Modify the optimisation flags for FastFixedDtoa.
This fixes a non-determinism issue in the asm produced for
this function when cross-compiling on x86_64 and aarch64 for
the arm-linux-gnueabihf HOST.

--- a/qtbase/src/3rdparty/double-conversion/fixed-dtoa.h
+++ b/qtbase/src/3rdparty/double-conversion/fixed-dtoa.h
@@ -48,9 +48,12 @@ namespace double_conversion {
 //
 // This method only works for some parameters. If it can't handle the input it
 // returns false. The output is null-terminated when the function succeeds.
+#pragma GCC push_options
+#pragma GCC optimize ("-O1")
 bool FastFixedDtoa(double v, int fractional_count,
                    Vector<char> buffer, int* length, int* decimal_point);
 
+#pragma GCC pop_options
 }  // namespace double_conversion
 
 #endif  // DOUBLE_CONVERSION_FIXED_DTOA_H_
